moveit_planners/trajopt/src/trajopt_planner_manager.cpp

```diff
@@ -37,104 +37,101 @@
 */
 
 #include <moveit/planning_interface/planning_interface.h>
-
 #include <moveit/collision_detection_fcl/collision_detector_allocator_fcl.h>
-
 #include <class_loader/class_loader.hpp>
-
 #include <trajopt_interface/trajopt_planning_context.h>
 
 namespace trajopt_interface
 {
-class TrajOptPlannerManager : public planning_interface::PlannerManager
-{
-public:
-  TrajOptPlannerManager() : planning_interface::PlannerManager()
-  {
-  }
-
-  bool initialize(const moveit::core::RobotModelConstPtr& model, const std::string& ns) override
+  class TrajOptPlannerManager : public planning_interface::PlannerManager
   {
-    ROS_INFO(" ======================================= initialize gets called");
-
-    if (!ns.empty())
-      nh_ = ros::NodeHandle(ns);
-    std::string trajopt_ns = ns.empty() ? "trajopt" : ns + "/trajopt";
-
-    for (const std::string& gpName : model->getJointModelGroupNames())
+  public:
+    TrajOptPlannerManager() : planning_interface::PlannerManager()
     {
-      ROS_INFO(" ======================================= group name: %s, robot model: %s", gpName.c_str(),
-               model->getName().c_str());
-      planning_contexts_[gpName] = std::make_shared<TrajOptPlanningContext>("trajopt_planning_context", gpName, model);
     }
 
-    return true;
-  }
+    bool initialize(const moveit::core::RobotModelConstPtr& model, const std::string& ns) override
+    {
+      ROS_INFO(" ======================================= initialize gets called");
 
-  bool canServiceRequest(const moveit_msgs::MotionPlanRequest& req) const override
-  {
-    return req.trajectory_constraints.constraints.empty();
-  }
+      if (!ns.empty())
+        nh_ = ros::NodeHandle(ns);
 
-  std::string getDescription() const override
-  {
-    return "TrajOpt";
-  }
+      std::string trajopt_ns = ns.empty() ? "trajopt" : ns + "/trajopt";
+      for (const std::string& gpName : model->getJointModelGroupNames())
```

Comment:
The naming convention for local variables is `snake_case`

```suggestion
      for (const std::string& group_name : model->getJointModelGroupNames())
```